test/rules/curly.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+	valid: [
+		"if (foo) {\n" +
```

Comment:
Should this instead be a long multiline template literal?

Reply:
Yup!